modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/admin/EventProcessorAdminServiceTestCase.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.integration.test.admin;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.event.processor.stub.types.ExecutionPlanConfigurationDto;
+import org.wso2.carbon.event.processor.stub.types.StreamDefinitionDto;
+import org.wso2.cep.integration.common.utils.CEPIntegrationTest;
+
+import java.io.File;
+
+public class EventProcessorAdminServiceTestCase extends CEPIntegrationTest {
+
+    private static final Log log = LogFactory.getLog(EventProcessorAdminServiceTestCase.class);
+    private static int eventStreamCount;
+    private static int executionPlanCount;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        super.init(TestUserMode.SUPER_TENANT_ADMIN);
+        String loggedInSessionCookie = getSessionCookie();
+
+        eventProcessorAdminServiceClient = configurationUtil.getEventProcessorAdminServiceClient(backendURL, loggedInSessionCookie);
+        eventStreamManagerAdminServiceClient = configurationUtil.getEventStreamManagerAdminServiceClient(backendURL, loggedInSessionCookie);
```

Comment:
@keizer619 
Thanks. Fixed for the above class in PR: https://github.com/wso2/product-cep/pull/186